<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerManagementForm</class>
 <widget class="QWidget" name="CustomerManagementForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="unassign_cust_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>205</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Unassign Customer</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/wnt/minus.png</normaloff>:/wnt/minus.png</iconset>
   </property>
  </widget>
  <widget class="QTableWidget" name="assigned_cust_twidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>235</y>
     <width>526</width>
     <height>141</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Address</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>House No</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>City</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CAN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tariff Category</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="assign_cust_button">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>205</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Assign Customer</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/wnt/plus.png</normaloff>:/wnt/plus.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>220</y>
     <width>136</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Assigned Customer(s)</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="registered_cust_twidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>80</y>
     <width>526</width>
     <height>121</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Address</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>House No</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>City</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CAN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tariff Category</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>65</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Registered Customer(s)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_term_edit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>176</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>15</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Term</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>27</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Add Damage</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/wnt/search.png</normaloff>:/wnt/search.png</iconset>
   </property>
  </widget>
  <widget class="QRadioButton" name="id_rbutton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ID / CAN</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="name_rbutton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>35</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
 </widget>
 <tabstops>
  <tabstop>id_rbutton</tabstop>
  <tabstop>name_rbutton</tabstop>
  <tabstop>search_term_edit</tabstop>
  <tabstop>search_button</tabstop>
  <tabstop>registered_cust_twidget</tabstop>
  <tabstop>assign_cust_button</tabstop>
  <tabstop>unassign_cust_button</tabstop>
  <tabstop>assigned_cust_twidget</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
